README.md

```diff
@@ -89,14 +89,14 @@ For the complete example, see
 public final class QuickStart {
   private static final Tagger tagger = Tags.getTagger();
   private static final ViewManager viewManager = Stats.getViewManager();
-  private static final StatsRecorder statsRecorder = Stats.getStatsRecorder();  
+  private static final StatsRecorder statsRecorder = Stats.getStatsRecorder();
```

Comment:
Instead of duplicating the code in the README, can we just add a hyperlink to the *.java file?

Reply:
The code example in README is slightly different from QuickStart.java though. QuickStart.java should be made executable so it has a main(), while README is more like a real class whose methods could be invoked multiple times.

Reply:
Ok, I'll leave it as-is.